microsoft-edge/devtools-guide-chromium/css/grid.md

```diff
@@ -26,30 +24,28 @@ CSS Grid is a powerful layout paradigm for the web.  A great place to get starte
 <!-- ====================================================================== -->
 ## Discover CSS grids
 
-When an HTML element on your page has `display: grid` or `display: inline-grid` applied to it, a `grid` badge is displayed next to it in the **Elements** tool:
+When an HTML element on your page has `display: grid` or `display: inline-grid` applied to it, a **grid** badge is displayed next to it in the **Elements** tool:
 
 ![Discover grid](./grid-images/grid-discover-grid.png)
 
-Click the badge to toggle the display of a grid overlay on the page.  The overlay appears over the element, laid out like a grid to display the position of the grid lines and tracks:
+Click the **grid** badge next to an element to display a grid overlay over that element in the rendered webpage.  The overlay appears over the element, and shows the position of the grid lines and tracks:
 
 ![Toggle grid badge](./grid-images/grid-highlight-grid.png)
 
-Open the **Layout** pane.  When grids are included on a page, the **Layout** pane includes a **Grid** section containing a number of options for viewing the grids.
+When grids are included on a page, the **Layout** pane in the **Elements** tool includes a **Grid** section that lists the grid containers and lets you configure the information which grid overlays should display.
 
 ![Layout pane](./grid-images/grid-layout-pane.png)
 
 The **Grid** section in the **Layout** pane contains the following 2 sub-sections:
 
-*  Overlay display settings
-*  Grid overlays
-
-<!--todo: verify the details for each of the sub-sections -->
+*  **Overlay display settings**, which lets you configure the information displayed in the grid overlay.
+*  **Grid overlays**, which lists the grid containers present on the webpage, and lets you enable or disable the corresponding grid overlays.
```

Comment:
```suggestion
*  **Overlay display settings** - Use this subsection to configure the information that's displayed in the grid overlay.
*  **Grid overlays** - Lists the grid containers that are present on the webpage. Use this subsection to show or hide the corresponding grid overlays.
```
one might wonder what corp style guide (it's end-user focused, not dev-focused) says about:
* lets you - https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/l/let-lets
   * allows you - https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/a/allow-allows
* enable - https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/e/enable-enables
* disable - https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/d/disable-disabled

@captainbrosset i might try table layout, which adds info in the header row, enabling tighter wording & faster skimming.

The **Grid** section in the **Layout** pane contains the following subsections:

| Subsection | Purpose |
| --- | --- |
| Overlay display settings | Configure which information is displayed in the grid overlay. |
| Grid overlays | Show or hide the grid overlay for each grid container that's present on the webpage. |